core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -131,9 +131,18 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
           //remove from resent tracking - it may get resent again, or get processed
           resent = None
         }
+        val kind = r.action.exec.kind
+        val memory = r.action.limits.memory.megabytes.MB
+
+        val prewarmedPoolForOtherKind = prewarmedPool.filter { info =>
+          info match {
+            case (_, PreWarmedData(_, `kind`, `memory`, _, _)) => false
```

Comment:
Why does it require filtering prewarm pool for other kinds?

Reply:
hm.. i forgot the reason why i want to `get the other kind/memory prewarmpool` ,
but then, i feel use `prewarmedPool` directly is more nature, already updated.